src/Type/Constant/ConstantStringType.php

```diff
@@ -61,8 +61,14 @@ function (): string {
 					return var_export($this->value, true);
 				}
 
+				try {
+					$truncatedValue = \Nette\Utils\Strings::truncate($this->value, self::DESCRIBE_LIMIT);
+				} catch (\Nette\Utils\RegexpException $e) {
+					$truncatedValue = substr($this->value, 0, self::DESCRIBE_LIMIT);
```

Comment:
We're also missing the ellipsis `…` here in case the string is longer than `self::DESCRIBE_LIMIT`.

Reply:
I added the ellipsis always because the exception is not thrown when the string is shorter than  `self::DESCRIBE_LIMIT`.

Reply:
Please add a test that short invalid UTF-8 is not truncated and ellipsis not added.